libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -48,6 +49,13 @@
  */
 #define NOTCHFILTER_ATTENUATION_CUTOFF 0.25
 
+#if APM_BUILD_TYPE(APM_BUILD_Heli)
+    // We cannot use throttle based notch on helis
+    #define NOTCHFILTER_DEFAULT_MODE 0 // fixed
+#else
+    #define NOTCHFILTER_DEFAULT_MODE 1 // throttle based
```

Comment:
Think this should be int8_t(HarmonicNotchDynamicMode::UpdateThrottle) etc

Reply:
All defaults are floats.